db/migrate/20160401043112_create_phones.rb

```diff
@@ -5,7 +5,7 @@ def change
 
       t.string :number, null: false
       t.string :extension
-      t.string :type, null: false
+      t.string :service_type, null: false
```

Comment:
Gotcha. If we prefer the "type" column name, we can go with the second suggestion:

>  overwrite <Model>.inheritance_column to use another column for that information.

`self.inheritance_column = '_unused'` or something like that in the `Phone` model should work around this without having to change the schema.